### PR TITLE
Ch08 cleanup

### PR DESCRIPTION
**Pull request by kagreenwell**
Cleanup of Chapter 8. Includes fixes for several issues opened for Chapter 8.

### CODE REVIEW

File: doc/ref_model/chapters/chapter08.md [hunk 1]
```diff
@@ -61,29 +61,30 @@ For the purposes of this chapter, NFVI+VNF testing will be performed for **Verif
 **Objective**<br>
 Perform NFVI+VNF Verification and Validations using CNTT reference architecture, leveraging the existing OPNFV and CVC Intake and Validation Process to onboard and validate new test projects for NFVI compliance.  Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
 
-**Scope**
+**Scope and Test Methodology**
 - Manifest Verifications (aka CVC Compliance) will ensure the NFVI is compliant, and delivered for testing, with hardware and software profile specifications defined by the Ref Model and Ref Architecture.
 - Empirical Validation with Reference Golden VNFs (aka CVC Validation) will ensure the NFVI runs with a set of VNF Families, or Classes, to minic production-like VNF connectivity, for the purposes of interopability checks.
 - Candidate VNF Validation (Validation & Performance) will ensure complete interopablity of VNF behavior on the NFVI leverage VVP/CVC test suites.  Testing ensures VNF can be spun up, modified, or removed, on the target NFVI (aka Interoperability).
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter08.md]: VVP/CVC -> VVP/VNFSDK

Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter08.md]: interopablity -> interoperability

**Comment by rabiabdel** [comment]
@mf4716 @kagreenwell would you mind please resolve @CsatariGergely reviews (commnets, questions) prior the meeting tomorrow?

File: doc/ref_model/chapters/chapter08.md (hunk 1 above)
Comment (mf4716) [reply doc/ref_model/chapters/chapter08.md]: Updated

Comment (mf4716) [reply doc/ref_model/chapters/chapter08.md]: Updated

**Comment by mf4716** [comment]
@rabiabdel @CsatariGergely @kagreenwell - completed minor changes from Gergely updated in Ch 8 doc

**Comment by kagreenwell** [comment]
All Botrange issues have been addressed and this pull request is ready to be reviewed on the master call and merged.

**Pull request merged**